src/desktop/auth-client.desktop

```diff
@@ -7,21 +7,15 @@ X-KDE-HasReadOnlyMode=true
 X-SuSE-YaST-Call=auth-client
 
 X-SuSE-YaST-Group=Net_advanced
-X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
 X-SuSE-YaST-AutoInst=all
-X-SuSE-YaST-Geometry=
-X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=auth-client
 X-SuSE-YaST-AutoInstClonable=true
-X-SuSE-YaST-AutoInstRequires=
-X-SuSE-YaST-AutoInstSchema=auth-client.rnc
-X-SuSE-YaST-Keywords=authentication,sssd,pam,ldap,krb5
+X-SuSE-YaST-Keywords=authentication,sssd,pam,ldap,kerberos,krb,krb5,windows,microsoft,ad,activedirectory
 
 Icon=yast-kerberos
 Exec=xdg-su -c "/sbin/yast2 auth-client"
 
-Name=Authentication Client
-GenericName=Configure the authentication client (sssd)
+Name=User Logon Management
```

Comment:
The two entry points, ["User Logon Management"](https://gist.github.com/mvidner/74c3e3811d3e6e4c18fdde766e10cefe#file-auth-client-1-initial-png) (yast2 auth-client) and ["LDAP and Kerberos Client"](https://gist.github.com/mvidner/74c3e3811d3e6e4c18fdde766e10cefe#file-ldapkrb-1-initial-png) (yast2 ldapkrb) have an identical initial screen, which is titled "System Authentication and Domain Configuration". That is very confusing to me.